packages/keystore/lib/db/migrations/20240920184658_initial_keystore_model.ts

```diff
@@ -0,0 +1,37 @@
+import type { Knex } from 'knex';
+import { PRIVATE_KEYS_TABLE } from '../../models/privatekeys.js';
+
+export const config = {
+    transaction: false
+};
+
+export async function up(knex: Knex): Promise<void> {
+    await knex.transaction(async (trx) => {
+        await trx.raw(`
+            CREATE TYPE entity_types AS ENUM (
+                'session',
+                'connection',
+                'environment'
```

Comment:
we might eventually want to use this package to handle environment private keys.